eng/workloads/workloads.csproj

```diff
@@ -74,8 +75,8 @@
       </ShortNames>
     </ItemGroup>
 
-    <MSBuild Projects="../dashboardpack/dashboardpack.csproj" Targets="restore;build" Properties="PackageOutputPath=$(PackageSource)" />
-    <MSBuild Projects="../dcppack/dcppack.csproj" Targets="restore;build" Properties="DcpRuntime=%(_DcpRuntimes.Identity);PackageOutputPath=$(PackageSource)" />
+    <MSBuild Projects="../dashboardpack/dashboardpack.csproj" Targets="restore;build" Properties="DashboardRuntime=%(_PackRuntimes.Identity);PackageOutputPath=$(PackageSource)" />
```

Comment:
Wait, don't we need to also call the publish target here?

Reply:
I don't think it is actually getting packed though, at least not locally so I may be doing something wrong. I pulled your branch and this is what I see:

![image](https://github.com/dotnet/aspire/assets/13854455/63975ac3-acbe-4b40-9eb5-4e07f2193ae7)

No tools folder.

Reply:
I just downloaded the one produced from your build, and that one doesn't have it either.

Reply:
I think I found the issue. It is a timing thing, we are defining None item statically in the dashboardpacks project, but the actual published assets don't get emitted until the BeforeBuild target is called. I will put up a PR to fix this.

Reply:
Good catch; it must have seemed like it was working locally for me due to a cached build from a previous run

Reply:
Yup. Put up PR https://github.com/dotnet/aspire/pull/1796 to fix this.